test/evp_test.c

```diff
@@ -485,8 +490,13 @@ static int cipher_test_parse(EVP_TEST *t, const char *keyword,
     if (strcmp(keyword, "Ciphertext") == 0)
         return parse_bin(value, &cdat->ciphertext, &cdat->ciphertext_len);
     if (cdat->aead) {
-        if (strcmp(keyword, "AAD") == 0)
-            return parse_bin(value, &cdat->aad, &cdat->aad_len);
+        if (strcmp(keyword, "AAD") == 0) {
```

Comment:
hm, should we have AAD1 AAD2 etc tags instead?

Reply:
That limits the number of AADs, I think it's easier to just allow multiple entries.